src/main/java/controllers/WorkoutStartController.java

```diff
@@ -0,0 +1,194 @@
+
+                    }
+                    newStartView.setExerciseNumber(newStartView.getExerciseNumber()+1);
+                    newStartView.setExerciseName(startModel.getWorkouts()[0].getWorkout().get(newStartView.getExerciseNumber()).getType());
+
+                    new WorkoutStartController(startModel, newStartView);
+                }
+
+            }
+
+            if (startView.getWorkoutNumber() == 2){
+
+                if (startModel.getWorkouts()[1].getWorkout().size()==startView.exerciseNumber){
+
+                    startView.dispose();
+                    WorkoutMenuView workoutMenuView = new WorkoutMenuView();
+
+                    workoutMenuView.setWorkoutNumber(2);
+
+                    new WorkoutMenuController(startModel, workoutMenuView);
+                }
+                else{
+
+                    startView.dispose();
+                    WorkoutStartView newStartView = new WorkoutStartView();
+
+                    newStartView.setWorkoutNumber(2);
+
+                    int workoutLength = startModel.getWorkouts()[1].getWorkout().size();
+                    for (int i = 0; i < workoutLength; i++){
+                        startView.addExercises(startModel.getWorkouts()[1].getWorkout().get(i).getType());
+
+                    }
+                    newStartView.setExerciseNumber(newStartView.getExerciseNumber()+1);
+                    newStartView.setExerciseName(startModel.getWorkouts()[1].getWorkout().get(newStartView.getExerciseNumber()).getType());
+
+                    new WorkoutStartController(startModel, newStartView);
+                }
+
+            }
+
+            if (startView.getWorkoutNumber() == 3){
+
+                if (startModel.getWorkouts()[2].getWorkout().size()==startView.exerciseNumber){
+
+                    startView.dispose();
+                    WorkoutMenuView workoutMenuView = new WorkoutMenuView();
+
+                    workoutMenuView.setWorkoutNumber(3);
+
+                    new WorkoutMenuController(startModel, workoutMenuView);
+                }
+                else{
+
+                    startView.dispose();
+                    WorkoutStartView newStartView = new WorkoutStartView();
+
+                    newStartView.setWorkoutNumber(1);
+
+                    int workoutLength = startModel.getWorkouts()[2].getWorkout().size();
+                    for (int i = 0; i < workoutLength; i++){
+                        startView.addExercises(startModel.getWorkouts()[2].getWorkout().get(i).getType());
+
+                    }
+                    newStartView.setExerciseNumber(newStartView.getExerciseNumber()+1);
+                    newStartView.setExerciseName(startModel.getWorkouts()[2].getWorkout().get(newStartView.getExerciseNumber()).getType());
+
+                    new WorkoutStartController(startModel, newStartView);
+                }
+
+            }
+
+            if (startView.getWorkoutNumber() == 4){
```

Comment:
I'm seeing many duplicates in the codes since for each workout, the code bodies are very similar. Improvements could be made by having less copy of the body and have it run for each workout depending on something that indicates which workout it should run for. Overall nice implementation.